### PR TITLE
Completed supporting sendInvoice method

### PR DESCRIPTION
**Pull request by NikitaKharkov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,6 +120,36 @@ $bot = new Bot('telegramToken');
...
+
+$sendInvoice = new SendInvoice(
+    $update->getMessage()->getChat()->getId(),
+    'Your title',
+    'Your description of invoice',
+    json_encode($payload), 
+    $providerToken,
+    '12314czasdq', // unique id
+    'UAH',
+    [['label' => 'PriceLabel', 'amount' => 3001]] // amount; here 30.01 UAH
```
Comment (makasim) [inline README.md]: use LabeledPrice here

File: src/SendInvoice.php [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        array $prices
+    ) {
+        set_value($this, 'chat_id', $chatId);
+        set_value($this, 'title', $title);
+        set_value($this, 'description', $description);
+        set_value($this, 'payload', $payload);
+        set_value($this, 'provider_token', $providerToken);
+        set_value($this, 'start_parameter', $startParameter);
+        set_value($this, 'currency', $currency);
+        set_objects($this, 'prices', $prices);
```
Comment (makasim) [inline src/SendInvoice.php]: introduce addPrice method and use it in the constructor to ensure type safety

Comment (makasim) [reply src/SendInvoice.php]: just like here https://github.com/formapro/telegram-bot-php/blob/master/src/InlineKeyboardMarkup.php#L22

File: src/SendInvoice.php [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    {
+        return get_value($this, 'start_parameter');
+    }
+
+    public function getCurrency(): string
+    {
+        return get_value($this, 'currency');
+    }
+
+    public function getPrices(): array
```
Comment (makasim) [inline src/SendInvoice.php]: add doc block `@return LabeledPrice[]`

File: src/SendInvoice.php (hunk 2 above)
Comment (makasim) [reply src/SendInvoice.php]: add an exception if prices are empty

File: src/SendInvoice.php [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    ) {
+        set_value($this, 'chat_id', $chatId);
+        set_value($this, 'title', $title);
+        set_value($this, 'description', $description);
+        set_value($this, 'payload', $payload);
+        set_value($this, 'provider_token', $providerToken);
+        set_value($this, 'start_parameter', $startParameter);
+        set_value($this, 'currency', $currency);
+
+        foreach ($prices as $price) {
```
[the remark this answers is not in the document]
Comment (makasim) [reply src/SendInvoice.php]: in the constructor. I would make set and add methods private

[the remark this answers is not in the document]
Comment (NikitaKharkov) [reply src/SendInvoice.php]: Aha... Okey!

**Pull request merged**